wrappers/python/nntile/__init__.py

```diff
@@ -11,5 +11,6 @@
 #
 # @version 1.0.0
 
-from . import functions, layer, loss, model, optimizer, pipeline, tensor
+from . import (functions, inference, layer, loss, model, optimizer, pipeline,
+               tensor)
```

Comment:
Why does top-level package import all inference stuff on `nntile` import?

Reply:
Because we provide this service to users

Reply:
What if user don't care about inference? Can user do `import nntile.inference` oneself?

Reply:
With this logic we can make init just empty. Users will import whatever they need.

Reply:
I believe this is a contradictory topic about design. I believe we should do it differently, as it could take some time to discuss

Reply:
> With this logic we can make init just empty. Users will import whatever they need.

What logic?

Reply:
Why do you want to hide high level API for inference from users? Because PyTorch does not have it, as I understand. But it does not mean we shall hide it, as this will become a part of NNTile.

Reply:
Sorry, but I believe it clearly discussed here. And for now seems to be very subjective
If you have a practical suggestion over design, please go on our meeting or lets hold different on small group

Reply:
It small topic. I guess we can discuss it here.

> Because PyTorch does not have it, as I understand.

First of all, do not do implicit assumptions and gaslight me, please.

Secondly, following to PyTorch design decision is actually far from a bad idea. They at least have a time-tested solution whilst you don't.

Thirdly, the reason why I proposed clean up top-level reimports is because of inference engine dependencies. They are abundant, massive, and diverse. Specifically, computation core of `nntile` requires only `numpy` whilst inference engine wants `uvicorn`. I don't care about `uvicorn`. I want to crunch a numbers and do not want massage JSONs. A user may share the same opinion.

Fourly, you should be careful about decision on public API. Bad decisions cause a lot of pain, are time consuming, might be irreversible. They are breaking and have side-effects. Just recall `nose` case. Basically, hiding internals behind `nntile._*` packages would be a good decision sometime.

Fively, necessary top-level imports make a package tightly coupled. It makes testing resourcefull, CI and packaging gets more complicated as well. This is obvious sign of poor design.

Finally, I think that you are not experienced enough and have already long record of questionable decisions for two years.

Thank you for merging this pull request and ignoring my valuable opinion.

Reply:
Valid thoughts about deps, you are right. It is how the conversation should be started, not from "Why do you do this?". Because the simplest answer will be "because". And probably it is what made you unhappy. And this is actually why your opinion is ignored here

Suggest better way to organize things. For now, I see you just don't like it and feel busy to suggest something. We are open to conversation and I will personally be happy to meet you. I believe you are very good developer) 
I am inviting you to our meetings and campus)

But keep in mind, that we should develop new functionality. And so where should be the balance between "lets discuss how to design api" and actually doing things. We are not oracles, so we are making mistakes in the design sometimes

Please don't say things like that. It is not good for reputation:

> Finally, I think that you are not experienced enough and have already long record of questionable decisions for two years.